source/wp-content/themes/wporg-developer-2023/templates/archive-command.html

```diff
@@ -1,4 +1,4 @@
-<!-- wp:template-part {"slug":"header-no-breadcrumbs","className":"has-display-contents"} /-->
```

Comment:
Looks like this pattern `header-no-breadcrumbs.html` is unused after these changes and can be removed

Reply:
Thanks. Done!